LongWarOfTheChosen/Src/LW_Overhaul/Classes/UIStrategyMapItem_AlienNetworkComponent_LW.uc

```diff
@@ -26,6 +29,7 @@ var UIProgressBar ProgressBar; // used to show remaining time for missions not y
 var int InDepth; // tracks depth of mouse in and outs, since it's recording from two possibly overlapping elements (2D and 3D icons)
 
 var localized string strRecon;
+var localized string InfiltrationTooltipString;

```

Comment:
Doesn't this need an entry in the localisation files?

Reply:
Correct.

If I already have it defined within LW_Overhaul.int as :
[UIStrategyMapItem_Mission_LW]
InfiltrationTooltipString="INFILTRATION"

Is there any way to pull the value from here, as in UIStrategyMapItem_Mission_LW, or do I need to make a separate entry :
[UIStrategyMapItem_AlienNetworkComponent_LW]
InfiltrationTooltipString="INFILTRATION"

Reply:
You need to add an extra entry. The other option is to just pull the localised string from the other class. It's a public `default` value after all.

Reply:
So I'm assuming :
class'UIStrategyMapItem_Mission_LW'.default.InfiltrationTooltipString

Reply:
Yes